tools/Dashboard/DevHome.Dashboard/Services/IWidgetExtensionService.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+
+namespace DevHome.Dashboard.Services;
+
+internal interface IWidgetExtensionService
+{
+    bool IsCoreWidgetExtension(string providerDefinitionId);

```

Comment:
I don't understand this name.  Is Core Widget Extension "what"?  Deleted, live, something else?  Or is the question "IS the core widget an extension? (Raised voice because I have no idea)

Reply:
It's an awkward place because we're talking about this *thing* both as a widget provider and a Dev Home extension and tying the two together. Looking at this method again, I didn't need to introduce that complexity to this particular method so I'm changing the name to IsCoreWidgetProvider (I think IsProviderACoreWidgetProvider is just too wordy) and between that plus a comment plus now the name correlating to the parameter name hopefully makes it clear enough.